fe/fe-core/src/main/java/org/apache/doris/resource/resourcegroup/ResourceGroup.java

```diff
@@ -85,11 +97,54 @@ private ResourceGroup(long id, String name, Map<String, String> properties, long
         }
     }
 
+    // called when first create a resource group, load from image or user new create a group
+    public void initQueryQueue() {
+        resetQueueProperty(properties);
+        // if query queue property is not set, when use default value
+        this.queryQueue = new QueryQueue(maxConcurrency, maxQueueSize, queueTimeout);
+    }
+
+    void resetQueryQueue(QueryQueue queryQueue) {
+        resetQueueProperty(properties);
+        this.queryQueue = queryQueue;
+        this.queryQueue.resetQueueProperty(this.maxConcurrency, this.maxQueueSize, this.queueTimeout);
+
+    }
+
+    private void resetQueueProperty(Map<String, String> properties) {
+        if (properties.containsKey(MAX_CONCURRENCY)) {
+            this.maxConcurrency = Integer.parseInt(properties.get(MAX_CONCURRENCY));
+        } else {
+            this.maxConcurrency = Integer.MAX_VALUE;
+            properties.put(MAX_CONCURRENCY, String.valueOf(this.maxConcurrency));
```

Comment:
Even if the user does not enable the query queue feature, he can still see the query queue related properties when using show resource groups, which may confuse the user.

Reply:
Concurrency is a necessary property for group, so it should be showed all the time.
Currently ```enable enable_query_queue``` is a  temporary variable, it should be keep until this feature is stable enough.